packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sequelize/core",
   "description": "Sequelize is a promise-based Node.js ORM tool for Postgres, MySQL, MariaDB, SQLite, Microsoft SQL Server, Amazon Redshift, Snowflake’s Data Cloud, Db2, and IBM i. It features solid transaction support, relations, eager and lazy loading, read replication and more.",
-  "version": "7.0.0-alpha.20",
+  "version": "0.0.0-development",
```

Comment:
lerna uses the version in package.json to determine the next version. When we merge this on the v7 branch, it is very important to make sure we don't actually replace this version number in the v7 branch.